frontend/app/templates/admin/users.hbs

```diff
@@ -1,5 +1,12 @@
 <div class="pt-3">
-  <h1>{{t "admin.users.index.title"}}</h1> 
+  <div class="justify-content-end">
+    <LinkTo @route="index">
+      <button type="button" class="close" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+    </LinkTo>
```

Comment:
Maybe style the back-button like on the account-show page. https://github.com/puzzle/cryptopus/blob/5e89e8b0fd472a01764d980a1825f1ee296a4027/frontend/app/templates/components/account/show.hbs#L10